src/main/java/eu/darkbot/utils/ArrayUtils.java

```diff
@@ -0,0 +1,18 @@
+package eu.darkbot.utils;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+public class ArrayUtils {
+
+    @SafeVarargs
+    public static <T> List<T> asImmutableList(T... a) {
+        return Collections.unmodifiableList(Arrays.asList(a));
+    }
+
+    public static boolean isEmpty(Collection<?> coll) {
+        return coll == null || coll.isEmpty();
+    }
```

Comment:
I am not convinced that this is a good idea, it can hide behaviour that may not be expected by whoever sees a call to "ArrayUtils.isEmpty" if they do not know what it is